content/page/what.en.md

```diff
@@ -77,12 +77,10 @@ If you are curious or want to help us to improve εxodus, you can file issues or
 
 ## εxodus Android Application
 
-Our application is designed **to list the applications already installed on your mobile phone** (both from [Google Play](https://play.google.com/store) or [F-Droid](f-droid.org/) and report the trackers embedded in each application.
+Our application is designed **to list the applications already installed on your mobile phone** (both from [Google Play](https://play.google.com/store) or [F-Droid](f-droid.org/)) and report the trackers embedded in each application.
```

Comment:
```suggestion
Our application is designed **to list the applications already installed on your mobile phone** (both from [Google Play](https://play.google.com/store) or [F-Droid](f-droid.org/) and report the trackers embedded in each application.
```

Looks like an unwanted change to me!

Reply:
It's not and unwanted change, a parenthesis is open and never close in your proposed change.